Bindings/bindings.json

```diff
@@ -474,6 +474,124 @@
             ]
         },
         {
+            "type": "apiHubTable",
+            "displayName": "SaaS Table",
+            "direction": "in",
+            "settings": [
+                {
+                    "name": "name",
+                    "value": "string",
+                    "defaultValue": "input",
+                    "required": true,
+                    "label": "Parameter name",
+                    "help": "This is the name of the parameter in the code for trigger to bind to.",
+                    "validators": [
+                        {
+                            "expression": "^[a-zA-Z_$][a-zA-Z_$0-9]*$",
+                            "errorText": "[variables('parameterName')]"
+                        }
+                    ]
+                },
+                {
+                    "name": "dataSetName",
+                    "value": "string",
+                    "defaultValue": "default",
+                    "required": false,
+                    "label": "Data Set Name",
```

Comment:
It looks like data set can be either two words or one word.  I personally feel like it should be two words so I'll leave it unless you feel strongly about it.